src/components/Header/Header.jsx

```diff
@@ -1,6 +1,14 @@
-// import the React library here
 // import a css file containig article styles
 
-// Create a Header function returning the HTML of header block
+import React from 'react';
+import Navigation from '../Navigation/Navigation';
```

Comment:
```suggestion
import Navigation from '../Navigation/Navigation.tsx';
```
Please check for errors before commiting. In most cases, they directly say what the problem is